config/locales/forms/zh.yml

```diff
@@ -109,16 +109,15 @@ zh:
       nickname: 设备昵称
       nickname_hint: 这样如果你添加了更多使用人脸或触摸解锁的设备的话，你就能把它们分辨开来。
     webauthn_setup:
-      intro: Use your physical security key to add an additional layer of protection
-        to your %{app_name} account to prevent unauthorized access.
+      intro: 使用实体安全密钥为您的 %{app_name} 帐户添加了额外一层安全保障，以防止未经授权的访问。了解有关安全密钥的更多信息
       learn_more: Learn more about security keys
       nickname: 安全密钥昵称
       saving: 正在保存你的凭据…
-      set_up: Set up security key
+      set_up: 设置安全密钥
```

Comment:
I think we need to remove exceptions for these in `i18n_spec.rb` `ALLOWED_UNTRANSLATED_KEYS`.

Ideally the specs will fail as-is because we're needlessly exempting.

Reply:
> Ideally the specs will fail as-is because we're needlessly exempting.

Yeah, it does fail:

```
  1) I18n does not have untranslated keys
     Failure/Error:
       expect(unused_allowed_untranslated_keys).to(
         be_empty,
         "unused allowed untranslated i18n keys: #{unused_allowed_untranslated_keys}",
       )
     
       unused allowed untranslated i18n keys: [{:key=>"forms.webauthn_setup.intro", :locales=>[:zh]}, {:key=>"forms.webauthn_setup.set_up", :locales=>[:zh]}, {:key=>"forms.webauthn_setup.step_2", :locales=>[:zh]}, {:key=>"forms.webauthn_setup.step_3", :locales=>[:zh]}, {:key=>"forms.webauthn_setup.step_3a", :locales=>[:zh]}]
```